modules/administration-guide/pages/configuring-che-storage-size.adoc

```diff
@@ -0,0 +1,38 @@
+:_content-type: CONCEPT
+:description: Configuring the storage size in {prod-short}
+:keywords: administration guide, configuring, {prod-id-short}, storage size, PVC size, pvc
+:navtitle: Configuring the storage size in {prod-short}
+:page-aliases: installation-guide:configuring-che-storage-size.adoc
+
+[id="configuring-{prod-id-short}-workspace-pvc-size"]
+= Configuring the storage size in {prod-short}
+
+The persistent volume claim (PVC) size can be configured when using the `per-user` or `per-workspace` storage strategies. PVC sizes in the Che Cluster Custom Resource must be specified in the format of a https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/[{kubernetes} resource quantity]. For more details on the available storage strategies, see xref:configuring-che-storage-strategy.adoc[]
+
+.Default persistent volume claim sizes:
```

Comment:
FYI, we have an [online guide about modular docs](https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs).

We reserve the `._<Heading>_` syntax for three sections in a procedure module:
`.Prerequisites` (only if any are included, otherwise omitted)
`.Procedure`
`.Additional resources`  (only if any are included, otherwise omitted)

So I suggest removing the `.`:
```suggestion
Default persistent volume claim sizes:
```